<launch>
  <rosparam file="$(find my_command_arm)/config/poses.yaml" command="load" />
  <node pkg="my_command_arm" type="ambil_data_node" name="ambil_data_node" output="screen"/>
</launch>
